docs/index.rst

```diff
@@ -207,20 +207,21 @@ This command will generate a ``PostFactory`` class that looks like this:
         /**
          * ...
          *
-         * @phpstan-method Post&Proxy create(array|callable $attributes = [])
-         * @phpstan-method static Post&Proxy createOne(array $attributes = [])
-         * @phpstan-method static Post&Proxy find(object|array|mixed $criteria)
-         * @phpstan-method static Post&Proxy findOrCreate(array $attributes)
-         * @phpstan-method static Post&Proxy first(string $sortedField = 'id')
-         * @phpstan-method static Post&Proxy last(string $sortedField = 'id')
-         * @phpstan-method static Post&Proxy random(array $attributes = [])
-         * @phpstan-method static Post&Proxy randomOrCreate(array $attributes = [])
-         * @phpstan-method static Post[]&Proxy[] all()
-         * @phpstan-method static Post[]&Proxy[] createMany(int $number, array|callable $attributes = [])
-         * @phpstan-method static Post[]&Proxy[] createSequence(array|callable $sequence)
-         * @phpstan-method static Post[]&Proxy[] findBy(array $attributes)
-         * @phpstan-method static Post[]&Proxy[] randomRange(int $min, int $max, array $attributes = [])
-         * @phpstan-method static Post[]&Proxy[] randomSet(int $number, array $attributes = [])
+         * @phpstan-method Proxy<Post> create(array|callable $attributes = [])
```

Comment:
I'm leaving this here, because if phpstan is used in a custom way (eg: like in this bundle :wink: ) we cannot detect it is used in the project

Reply:
👍🏻